django_sis/settings.py

```diff
@@ -488,7 +488,56 @@ def skip_unreadable_post(record):
     'APPLICANT_EMAIL_ALERT' : (False, "Send email alert on applicant submission"),
     'APPLICANT_EMAIL_ALERT_ADDRESSES' : ('',
         "Email addresses to send alert to; only one email address per line"),
-    'FROM_EMAIL_ADDRESS' : ('', "Default email address to use for sending mail")
+    'FROM_EMAIL_ADDRESS' : ('', "Default email address to use for sending mail"),
+    'SCHOOL_PAY_RATE_PER_HOUR': ('13.00', ''),
+    'WORK_STUDY_SHOW_COMMENT_DEFAULT': ('True', ''),
+    'WORK_STUDY_TIMESHEET_INITIAL_TIME': ('True', ''),
+    'WORK_STUDY_CONTRACT_FROM_ADDRESS': ('donotreply@cristoreyny.org', ''),
+    'WORK_STUDY_CONTRACT_CC_ADDRESS': ('', ''),
+    'STUDENTS_PER_FTE': ('.2', ''),
+    'S': ('5', 'David is an idiot who can not code'),
+    'EDIT_ALL_STUDENT_WORKER_FIELDS': ('False', ''),
+    'COUNSELING_REFERRAL_NOTICE_EMAIL_TO': ('', ''),
+    'ADMISSIONS_TO_STUDENT_ALSO_MAKES_STUDENT_WORKER': ('False', ''),
+    'ADMISSIONS_OVERRIDE_YEAR_START': ('', 'Must be ISO date (ex 2012-10-25) or blank'),
+    'PASSING_GRADE': ('70', ''),
+    'LETTER_PASSING_GRADE': ('A,B,C,P', ''),
+    'ONLY_ACTIVE_CLASSES_IN_SCHEDULE': ('', ''),
+    'HIDE_EMPTY_PERIODS_IN_SCHEDULE': ('False', ''),
+    'ATTENDANCE_CREATE_WORK_ATTENDANCE': ('False', ''),
+    'DEFAULT_CITY': ('', ''),
+    'HOW_TO_OBTAIN_STUDENT_EMAIL': ('append','append, user, or student', ''),
+    'EMAIL': ('@change.me', ''),
+    'CLEAR_PLACEMENT_FOR_INACTIVE_STUDENTS': ('False', ''),
+    'BENCHMARK_BASED_GRADING': ('False', ''),
+    'VOLUNTEER_TRACK_REQUIRED_HOURS': ('20', ''),
+    'VOLUNTEER_TRACK_MANAGER_EMAILS': ('', ''),
+    'ATTENDANCE_DISC_TARDIES_BEFORE_DISC': ('1', ''),
+    'ATTENDANCE_DISC_INFRACTION': ('', ''),
+    'ATTENDANCE_DISC_ACTION': ('', ''),
+    'DISCIPLINE_MERIT_DEFAULT_DAYS': ('14', ''),
```

Comment:
You added all this but I'm guessing only support 1 or 2 right? This is going to be highly confusing to users and even to ourselves. Can you remove all but the one or two that are supported? Half done commits are deadly. 

I also don't see the removed of these from Configuration. Do one small thing right instead of 50 half dones.